### PR TITLE
refactor rabbitmq

### PR DESCRIPTION
**Pull request by belwase**
- rabbitmq refactor
- removed upwork-devs folder

### CODE REVIEW

File: rabbitmq_receiver/receiver.py [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
+            if not handler:
+                logger.info("invalid payload type.")
+            handler(payload)
 
         logger.info(" [x] Done")
         ch.basic_ack(delivery_tag=method.delivery_tag)
 
-    channel.basic_qos(prefetch_count=1)
-    channel.basic_consume(queue=os.environ["MQ_QUEUE"], on_message_callback=callback)
+    def connect(self):
```
Comment (susilupwork) [inline rabbitmq_receiver/receiver.py:43]: This method needs Exception handling still.

File: rabbitmq_receiver/receiver.py [hunk 2]
```diff
@@ -1,43 +1,78 @@
...
+        self.connection = pika.SelectConnection(
+            parameters=pika.ConnectionParameters(host=self.host),
+            on_open_callback=self.on_connection_open,
+            on_open_error_callback=self.on_connection_open_error,
+            on_close_callback=self.on_connection_closed,
+        )
+        return self.connection
+
+    def on_connection_open(self, _unused_connection):
+        logger.info("Connection opened")
```
Comment (susilupwork) [inline rabbitmq_receiver/receiver.py:54]: This log should be debug.

**Review by susilupwork — APPROVED** [review]
Very good. Please add some INFO logging  with little more information e.g. .info(msg), which helps in prod scenario.

**Comment by belwase** [comment]
@DinisCruz  I've removed upwork-devs folder since that's not being used anymore and it's content are already on top-level.

**Pull request merged**